src/quo2/components/bottom_sheet/view.cljs

```diff
@@ -0,0 +1,218 @@
+(ns quo2.components.bottom-sheet.view
+  (:require [reagent.core :as reagent]
+            [quo.animated :as animated]
+            [quo.react-native :as rn]
+            [quo.react :as react]
+            [quo.platform :as platform]
+            [cljs-bean.core :as bean]
+            [quo.components.safe-area :as safe-area]
+            [quo2.components.bottom-sheet.style :as styles]
+            [quo.gesture-handler :as gesture-handler]
```

Comment:
if we want to move it , it should use only quo2 components

Reply:
Should we also copy gesture-handler, safe-area, etc. to quo2 namespace? Or are/will there be other libraries in quo2 for this?

I didn't intent to rewrite the whole bottom_sheet, just copy over so i can adjust styling to the new design.

What do you think?

Reply:
Ofc, i can replace quo(1) colors with the new ones, but i meant for the other components

Reply:
hey @chrispader sorry for the late response now we have all we need in the new code structures, so reanimated ,safe-area, and gesture now in react-native root package so they should be used